src/ixdat/plotters/base_mpl_plotter.py

```diff
@@ -0,0 +1,53 @@
+from matplotlib import pyplot as plt
+from matplotlib import gridspec
+
+
+class MPLPlotter:
+    def new_ax(self, xlabel=None, ylabel=None):
+        """Return a new matplotlib axis optionally with the given x and y labels"""
+        fig, ax = plt.subplots()
+        if xlabel:
+            ax.set_xlabel(xlabel)
+        if ylabel:
+            ax.set_ylabel(ylabel)
+        return ax
+
+    def new_two_panel_axes(self, n_bottom=1, n_top=1, emphasis="top"):
+        """Return the axes handles for a bottom and top panel.
+
+        Args:
+            n_top (int): 1 for a single y-axis, 2 for left and right y-axes on top panel
+            n_bottom (int): 1 for a single y-axis, 2 for left and right y-axes on bottom
+            emphasis (str or None): "top" for bigger top panel, "bottom" for bigger
+                bottom panel, None for equal-sized panels
+
+        Returns list of axes: top left, bottom left(, bottom right)(, top right)
```

Comment:
Yes, I gathered that the reason was something like that.

> If not, then the method will still have a variable length of the return...

Yes. What I was proposing was to still have variable length of the return value, basically what I suggested in my comment `top left, bottom left(, top right, bottom right)`, only that there would only be 2 forms instead of three and the order or the axis could then be something more systematic.

Reply:
So, I think you're right. The reason I hesitate is just that the change should propogate to the axes lists returned by the functions in `ECMSPlotter`, so everyone who has used ixdat to plot EC-MS data or spectroelectrochemistry data will have to change `axes[2]` to `axes[3]` when they set ylim or ylabel or whatever for the electrode current axis in their old scripts.

Reply:
Uh, that might hurt. I will leave the timing up to you. I think, when we do it, we would have to describe it as a backwards compatability problem in the release notes and I will try and hand-hold it though with the people here.

Reply:
Here goes!
According to Semantic versioning 2.0.0 item 4, we can change whatever we want whenever we want before ixdat v1.0.0
https://semver.org/#spec-item-4

Reply:
Done.